### PR TITLE
gimp2: update to 2.10.36

### PR DESCRIPTION
**Pull request by i0ntempest**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by mascguy** [comment]
Let's update `gimp2-devel` only at this point, as we need to validate the build across all macOS releases.

I'll take care of updating the rest, once we're sure there aren't any issues.

**Comment by i0ntempest** [comment]
> Let's update `gimp2-devel` only at this point, as we need to validate the build across all macOS releases.
> 
> I'll take care of updating the rest, once we're sure there aren't any issues.

Did that [here](https://github.com/macports/macports-ports/pull/21577). I'll leave this open, you can use this or close if it's ultimately not needed.

File: aqua/gimp2-launcher/files/patch-Info.plist.diff [hunk 1]
```diff
@@ -13,7 +13,7 @@
      <key>NSHumanReadableCopyright</key>
      <string>© 2001-2022 The GIMP Team</string>
      <key>LSMinimumSystemVersion</key>
--	  <string>10.12</string>
+-	  <string>10.13</string>
```
Comment (mascguy) [inline aqua/gimp2-launcher/files/patch-Info.plist.diff:16]: Just curious, why is this needed?

Comment (i0ntempest) [reply aqua/gimp2-launcher/files/patch-Info.plist.diff:16]: Because the os version requirement of the app bundle changed. But since we build our own binary maybe we should remove this key altogether.

**Comment by mascguy** [comment]
Converting to draft, until I've had a chance to test locally.

**Comment by reneeotten** [comment]
Ping @mascguy

**Comment by mascguy** [comment]
> Ping @mascguy 

Haven't forgotten about it, hoping to find some time over the holidays

**Comment by mascguy** [comment]
> Converting to draft, until I've had a chance to test locally.

It looks like the original builds failed for 10.15 and earlier, due to OpenBLAS being broken at the time.

The latter now appears fixed, so re-queued those failed builds. More to follow, once the buildbots are done.

**Review by mascguy — APPROVED** [review]

**Comment by mascguy** [comment]
Sorry for the delay in reviewing. And thank you Zhenfu!
